packages/likert/configure/src/main.jsx

```diff
@@ -0,0 +1,377 @@
+  const { classes, model, onChangeModel } = props;
+  const onChangeLikertScale = (e) => {
+    const likertScale = e.target.value;
+    onChangeModel({
+      ...model,
+      likertScale,
+      choices: generateChoices(likertScale, model.likertType)
+    });
+  };
+  return (
+    <div className={classes.radioButtonsWrapper}>
+      <p className={classes.radioButtonsColumnHeader}>
+        Likert Scale
+      </p>
+      <RadioGroup
+        aria-label='likertScale'
+        name='likertScale'
+        value={model.likertScale}
+        onChange={onChangeLikertScale}
+      >
+        <FormControlLabel value={LIKERT_SCALE.likert3} control={<Radio/>} label='Likert 3'/>
+        <FormControlLabel value={LIKERT_SCALE.likert5} control={<Radio/>} label='Likert 5'/>
+        <FormControlLabel value={LIKERT_SCALE.likert7} control={<Radio/>} label='Likert 7'/>
+      </RadioGroup>
+    </div>
+  )
+});
+
+const LikertType = withStyles(styles)((props) => {
+  const { classes, model, onChangeModel } = props;
+  const onChangeLikertType = (e) => {
+    const likertType = e.target.value;
+    onChangeModel({
+      ...model,
+      likertType,
+      choices: generateChoices(model.likertScale, likertType)
+    });
+  };
+  return (
+    <div className={classes.radioButtonsWrapper}>
+      <p className={classes.radioButtonsColumnHeader}>
+        Label Type
+      </p>
+
+      <div className={classes.flexRow}>
+        <RadioGroup
+          aria-label='likertLabelType'
+          name='likertLabelType'
+          value={model.likertType}
+          onChange={onChangeLikertType}>
+          <FormControlLabel value={LIKERT_TYPE.agreement} control={<Radio/>} label='Agreement'/>
+          <FormControlLabel value={LIKERT_TYPE.frequency} control={<Radio/>} label='Frequency'/>
+          <FormControlLabel value={LIKERT_TYPE.yesNo} control={<Radio/>} label='Yes/No'/>
+        </RadioGroup>
+
+        <RadioGroup
+          aria-label='likertLabelType'
+          name='likertLabelType'
+          value={model.likertType}
+          onChange={onChangeLikertType}>
+          <FormControlLabel value={LIKERT_TYPE.importance} control={<Radio/>} label='Importance'/>
+          <FormControlLabel value={LIKERT_TYPE.likelihood} control={<Radio/>} label='Likelihood'/>
+          <FormControlLabel value={LIKERT_TYPE.like} control={<Radio/>} label='Like'/>
+        </RadioGroup>
+      </div>
+    </div>
+  )
+});
+
+const Design = withStyles(styles)(props => {
+  const {
+    classes,
+    model,
+    configuration,
+    onPromptChanged,
+    onChoiceChanged,
+    imageSupport,
+    onChangeModel,
+    onConfigurationChanged,
+    onTeacherInstructionsChanged
+  } = props;
+  const {
+    prompt = {},
+    teacherInstructions = {},
+    studentInstructions = {},
+    scoringType = {},
+  } = configuration || {};
+  const {
+    teacherInstructionsEnabled
+  } = model || {};
+
+  const valuesMap = model.choices.reduce((acc, choice) => {
```

Comment:
This needs to be moved outside and made into a reusable utility function.

Reply:
solved